packages/ui/views/account/index.tsx

```diff
@@ -2,10 +2,11 @@ import React, { Component } from 'react'
 
 import Engine from 'lndr/engine'
 
-import { Text } from 'react-native'
+import { View, Text } from 'react-native'
 
 import Button from 'ui/components/button'
 import Tabs, { Tab } from 'ui/components/tabs'
+import ActionButton from 'ui/components/action'
```

Comment:
Inconsistent class name and folder name - make sure it's either `Action` or `ui/components/action-button`